src/carousel/carousel.scss

```diff
@@ -114,9 +118,13 @@
       transform: scale(0);
     }
 
-    &.iui-active::after {
-      @include themed {
-        background-color: t(iui-color-foreground-primary);
+    &.iui-active {
+      &::after {
```

Comment:
why syntax changed here?

Reply:
I tried a couple of different ways of updating the active indicator.  I think this change is leftover from a failed attempt.  Reverting.